test/scale-infra/INSTRUCTIONS.md

```diff
@@ -0,0 +1,100 @@
+### Examples of Individual Steps
+As an alternative to the quick-start deployment bootstrapping instructions, you can also run the individual steps starting from a local directory containing the git repo as follows:
+
+1. Create the KubeStellar core infra on AWS. The infrastructure includes a VPC, security groups, EC2 instances, etc.
+
+    ```bash
+    cd test/scale-infra
+    ansible-playbook deploy_vpc_core.yaml -e "region=us-east-2 name=<aws-iam-user>"
+    ansible-playbook create-ec2.yaml -e "cluster_name=core region=us-east-2 vpc_name=name=<vpc_name> aws_key_name=mykey  num_masters=1 num_workers=2 instance_type=t2.xlarge arch=x86_64 ec2_image='<your-aws-ami>'"
```

Comment:
Do I really want `name=` twice in `vpc_name=name=<vpc_name>` ?

Reply:
Testing confirms that I do NOT want `name=` in there twice.

Reply:
oh, that was a typo - fixed. Thanks!